static/src/javascripts/projects/common/modules/commercial/acquisitions-ophan.js

```diff
@@ -39,22 +39,27 @@ export const submitViewEvent = (componentEvent: ComponentEventWithoutAction) =>
         action: 'VIEW',
     });
 
+export const addReferrerData = (acquisitionData: {}) =>
+    // Note: the current page is the referrer data in the context of the acquisition.
+    Object.assign({}, acquisitionData, {
+        referrerPageviewId: config.get('ophan.pageViewId') || undefined,
```

Comment:
If you don't specify a second parameter to `config.get(...)`, it will return `undefined` by default. So this line is equivalent to:
```js
referrerPageviewId: config.get('ophan.pageViewId')
```